packages/next-app/src/app/internalmanager/[network]/[operationKind]/[tokenAddress]/page.tsx

```diff
@@ -76,28 +82,32 @@ export default function Page({
     switch (operationKind) {
```

Comment:
why did you create this switch case? any benefits you see? I personally prefer to do this using enum + dict/record